src/main/resources/config/application-dev.yml

```diff
@@ -3,6 +3,7 @@ faf-api:
   jwt:
     secretKeyPath: ${JWT_PRIVATE_KEY_PATH:test-pki-private.key}
     publicKeyPath: ${JWT_PUBLIC_KEY_PATH:test-pki-public.key}
+    hydraJwksUrl: ${JWT_HYDRA_JWKS_URL:https://hydra.test.faforever.com/.well-known/jwks.json}
```

Comment:
is hydra somehow relevant in this context? isn't it implementation detail?

Reply:
It's not necessarily Hydra that is relevant here, but it is the OpenID Connect token that we customized (manual extension for our role system). So it's not just a default Jwks url, it must point to a faf hydra instance.

Reply:
Slightly renamed it to emphasize it's faf hydra not any hydra

Reply:
I still don't understand, if we change hydra for octa or something else, wouldn't we need just to change the value and not the key?